corehq/apps/domain/decorators.py

```diff
@@ -314,6 +315,11 @@ def _inner(request, domain, *args, **kwargs):
 
 
 def _two_factor_required(view_func, domain, couch_user):
+    require_2fa = False
+    if toggles.TWO_FACTOR_SUPERUSER_ROLLOUT.enabled(couch_user.username):
+        require_2fa = True
+    if require_2fa:
+        return not getattr(view_func, 'two_factor_exempt', False)
```

Comment:
can you just stick this in the previous `if` statement? I'm not sure what the `require_2fa` variable adds here.

Reply:
Moved around here:
https://github.com/dimagi/commcare-hq/pull/19570/commits/9ba1f0dba28bb6befaf561594655cadebf644f48